rust_dev_preview/s3/README.md

```diff
@@ -1,20 +1,28 @@
-# AWS SDK for Rust code examples for Amazon S3
+# Amazon S3 examples using SDK for Rust
 
-## Purpose
+# Overview
 
-These examples demonstrate how to perform several Amazon Simple Storage Service 
+These examples demonstrate how to perform several Amazon Simple Storage Service
 (Amazon S3) operations using the developer preview version of the AWS SDK for Rust.
 
 *Amazon S3 is storage for the internet. You can use Amazon S3 to store and retrieve any
 amount of data at any time, from anywhere on the web.*
 
-## Code examples
+## ⚠ Important
 
-### Scenario examples
+- We recommend that you grant this code least privilege,
+  or at most the minimum permissions required to perform the task.
+  For more information, see
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
+  in the AWS Identity and Access Management User Guide.
+- This code has not been tested in all AWS Regions.
+  Some AWS services are available only in specific
+  [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+- Running this code might result in charges to your AWS account.
```

Comment:
- Running this code might result in charges to your AWS account.
- Running the tests might result in charges to your AWS account.
- We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
- This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).

Reply:
Fixed.